packer-plugin-amazon.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: packer-plugin-amazon
+  version: 1.3.2
+  epoch: 0
+  copyright:
+    - license: MPL-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/hashicorp/packer-plugin-amazon
+      tag: v${{package.version}}
+      expected-commit: 28aaa72ba200fffc90b9e64d8a89708926328ce8
+
+  - uses: go/bump
+    with:
+      deps: github.com/go-jose/go-jose/v3@v3.0.3 github.com/hashicorp/go-retryablehttp@v0.7.7 golang.org/x/crypto@v0.17.0 golang.org/x/net@v0.23.0 google.golang.org/protobuf@v1.33.0 github.com/hashicorp/packer-plugin-sdk@v0.5.4
+      replaces: github.com/go-jose/go-jose/v3=github.com/go-jose/go-jose/v3@v3.0.3 gopkg.in/square/go-jose.v2=github.com/go-jose/go-jose/v3@v3.0.3
+
+  - uses: go/build
+    with:
+      packages: .
+      output: packer-plugin-amazon
+
```

Comment:
Does the binary respond to a --version or --help? Given it's a go binary it's highly likely, and we could at least add a basic test

Reply:
Hi @ kranurag7, is this PR still needed?

Reply:
Have we been able to do any manual validation that the plugins are detected and usable by packer?